examples/demo/site/themes/bootstrap/modules/login/templates/block_login.tpl

```diff
@@ -0,0 +1,34 @@
+ 	<div class="primary-tabs">
+ 		{unless isset="$user"}
+			<h3>Login or <a href="/account/roc-register">Register</a></h3>
+		<div>
+			<div>	
+			    <form action method="POST">
+					<div>
+						<input type="text" name="username" required>
+						<label>Username</label>
+					</div>
+										
+					<div>
+						<input  type="password" name="password" required>
+						<label>Password</label>
+					</div>
+        
+					<button type="button" onclick="ROC_AUTH.login();">Login</button>
+				</form>
+			</div>
+    	</div>
+		<div>
+			<div>
+				<p>
+					<a href="/account/new-password">Forgot password?</a>
```

Comment:
it should use the equivalent of {$site_url/} ... otherwise we will have trouble to host the cms under a specific folder.